source/ed/types.cpp

```diff
@@ -114,6 +114,10 @@ bool Line::operator<(const Line& other) const {
     }
 }
 
+bool LineGroup::operator<(const LineGroup& other) const {
+    return this->name < other.name || (this->name == other.name && this < &other);
+}
```

Comment:
I don't like comparing pointers. As LineGroup is a Header, is has an idx on which we can compare instead of the address (which is not the same accross different run).

I prefere writing my lexical comparator more closely to the corresponding order:
```c++
bool LineGroup::operator<(const LineGroup& other) const {
    if (this->name != other.name) { return this->name < other.name; }
    return this->idx < other.idx;
}
```